src/modules/sync/diff.ts

```diff
@@ -117,3 +130,55 @@ const createDiffModel = <Entity extends Readonly<{ codename: string }>>(
       },
       { added: [], updated: new Map(), deleted: new Set() },
     );
+
+const getLanguageDiffModel = (params: DiffParams) => {
```

Comment:
What about adding a unit test on the whole `diff` function that would test that the change of the default language's codename is handled properly here?

Reply:
Also, you don't need to accept the whole `DiffParams` here. Just the source and target languages.